### ISSUE TITLE
MSAA is not applied in LWJGL3 when new window is created

### ISSUE THREAD

**Issue by fgnm** [libgdx/libgdx#6207]
#### Issue details
According to that lines : https://github.com/libgdx/libgdx/blob/5b4aa04c73d995c40932348bd7e16f0f30e82982/backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Application.java#L428-L436 when a new LWJGL3 window is create all of that `glfwWindowHint` are not set. Indeed, only the very first main window (that has `sharedContextWindow = 0`) will receive that instructions, but newly created ones doesn't.

This means that MSAA samples are applied only to the first window.

I've tried to force the `sharedContextWindow == 0` to `true` with a debugger to get `glfwWindowHint` executed even on other windows and MSAA is finally applied properly.

#### Reproduction steps/code
```Java
import com.badlogic.gdx.*;

public class Barebones extends ApplicationAdapter {
	public void create () {
		Lwjgl3Application app = (Lwjgl3Application) Gdx.app;
                Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
                config.setBackBufferConfig(8,8,8,8,16,8,16);
                app.newWindow(new Barebones2(), config);
	}

	public void render () {
		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
		// your code here
	}

	public static void main (String[] args) throws Exception {
                Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
                config.setBackBufferConfig(8,8,8,8,16,8,16);
		new LwjglApplication(new Barebones(), config);
	}
}
```
In this case the main first window has correctly applied MSAA while the second one not.

In my case the difference is highly appreciable, this is a screen of my application of a secondary window:
![immagine](https://user-images.githubusercontent.com/5543339/95475853-f2e08480-0986-11eb-8935-3a7cfc34699e.png)

This is instead how it looks if rendered on the first main window:
![immagine](https://user-images.githubusercontent.com/5543339/95475666-c2004f80-0986-11eb-9462-329bf0b01de2.png)

This control (`if(sharedContextWindow == 0)`) was introduced in this commit https://github.com/libgdx/libgdx/commit/feaec52ad636203db972f1fd85ceb03c4ab97aff#diff-ca97410d98b52757fa0955458cb0903aR331 but without a clear explanation of why it is necessary, apparently removing it seems to fix the issue.

#### Version of LibGDX and/or relevant dependencies
libGDX v1.9.11

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [X] Windows
- [X] Linux
- [X] MacOS

**Comment by mgsx-dev** [comment]
@badlogic do you remember the reason of this change? maybe it was necessary at this time and GLFW behavior changed since then?

**Comment by LengendsNeverDie** [comment]
Did you check to make sure there isn't any more slowness as it reacts to the command? There possibly could be a bug where you ask for it to do something and it has another action.

**Comment by fgnm** [comment]
Well no, in this case the instructions are completely skiped for newly create windows, I've made a PR to discuss about this, as the fix is pretty simple, just remove the wrong if condition :)

**Comment by crykn** [comment]
Closed in 8f3488375f8663fed4644ab9df6b461a9a4f5faf.
